logger.go

```diff
@@ -0,0 +1,115 @@
+/*
+
+This package implements our standard Logrus + Logentries + Sentry configuration
+
+If handles logrus initialization from environment variables.
+
+You will use it exactly as you would use logrus.
+
+
+Examples
+
+Log an info:
+
+	WithFields(logrus.Fields{
+		"driverId": driverId,
+		"rideId": rideId,
+	}).Info("Ride accepted")
+
+Log an info with a single field:
+
+	WithField("userId", userId).Info("User logged in")
+
+Debug, Warning, Error, etc... work exactly the same.
+
+
+Configuration
+
+SENTRY_DSN: If provided, warning and error logs will be sent to sentry.
+
+LOGGER_NAME - not yet implemented - The name of the logger.
+
+LOGGER_LEVEL - not yet implemented - The minimum level of the message to be actually logged.
+Possible values: "debug", "info", "warning", "error".
+
+LOGENTRIES_TOKEN - not yet implemented - If provided, logs will be sent to logentries.
+
+LOGSTASH_HOST - not yet implemented - If provided, logs will be sent to logstash.
+
+LOGSTASH_PORT - not yet implemented - Mandatory if LOGSTASH_HOST is provided.
+
+
+Notes
+
+Methods that allow logging without context are not provided, in order to discourage logging without context.
+
+ */
+package logger
+
+import (
+	"github.com/Sirupsen/logrus"
+	"os"
+	"github.com/evalphobia/logrus_sentry"
+	"time"
+)
+
+var logger *logrus.Logger
+
+/*
+Returns a ready to use instance of logrus.Logger, according to the configuration.
+
+Note: this instance is cached, so if the environment changes, you will need to call ReloadConfiguration() before
+calling this method.
+ */
+func GetLogger() *logrus.Logger {
+	if logger == nil {
```

Comment:
:+1: Will make the code clearer.
But I still need to test the `WithField` and `WithFields` methods, which depend on the singletton, so I still need `ReloadConfiguration`